app/js/collections/todos.js

```diff
@@ -0,0 +1,10 @@
+define([
+  'backbone',
+  'models/todo'
+], function (Backbone, ToDo) {
+  var ToDoCollection = Backbone.Collection.extend({
+    model: ToDo
+  });
+
+  return new ToDoCollection();
```

Comment:
Not good, by creating a new instance here you are making the collection a singleton, I will advise against it as  in the long run it will promote anti patterns like ability to change the collection from other parts of the app without any control.